src/main/app/claims/responseModelConverter.ts

```diff
@@ -332,78 +332,78 @@ export class ResponseModelConverter {
       incomes.push({
         type: IncomeType.JOB,
         frequency: income.salarySource.schedule.value as PaymentFrequency,
-        amountReceived: income.salarySource.amount
+        amount: income.salarySource.amount
       })
     }
     if (income.universalCreditSource && income.universalCreditSource.populated) {
       incomes.push({
         type: IncomeType.UNIVERSAL_CREDIT,
         frequency: income.universalCreditSource.schedule.value as PaymentFrequency,
-        amountReceived: income.universalCreditSource.amount
+        amount: income.universalCreditSource.amount
```

Comment:
This will require change in `claim-store` to reflect the rename, otherwise response submission will fail due to 400.

Reply:
created one for you: https://github.com/hmcts/cmc-claim-store/pull/446